### ISSUE TITLE
[Redesign] Clean up report page

### ISSUE THREAD

**Issue by hidde** [w3c/wcag-em-report-tool#372]
Let's simplify the report page and reduce complexity in these ways. There's a screenshot below to illustrate. cc @nitedog 

### Change title from “Website Accessibility Evaluation Report” to “Report”

Rationale: 

* “website“ is overly specific, one could create a report for a “web app”
* “website” is not needed, as Web is the W in WCAG
* ”Accessibility“ is not needed, it is the A in WCAG
* “Evaluation” is not needed, it is what the tool does

###  Remove intro paragraph

Rationale: 

* It explains what the buttons are for, but the buttons are and ought to be self-explanatory.  
* If people really struggle finding out what the buttons are for, they can try them

### Remove box around buttons

Rationale: 

* Consistency with ATAG Report Tool
* Makes UI more complex

### Use primary button for both actions

Rationale:

* Consistency with ATAG Report Tool
* JSON and HTML export are both same level of functionality, there is no hierarchy

### Use “Download Report” a button text

instead of “Download the evaluation report (HTML)” and “Save the evaluation data (JSON)” 

Rationale: 

* distinction between download and save is confusing, in either case you'd be downloading AND saving
* ultimately, it is the report you're downloading, “Download Report” is the least confusing way to describe that action
* consistency with ATAG Report Tool
* if people don't know that JSON means data, that's ok, they are unlikely to need the report in that data format
* we want to capitalise “Report” for consistency with other WAI properties

### Rename “Website Accessibility Evaluation Report” to “About the Evaluation” and make it h2

Rationale

* Consistency with ATAG Report Tool
* More descriptive heading for the section
* Better hierarchy

### Rename “Summary of the evaluation findings“ to “Executive Summary”

* Consistency with ATAG Report Tool
* Less jargony (or at least this jargon is less specific to our use case and more broadly used and understood)

### Use columns for key/value pairs, and bold values (rather than keys)

(columns only on screen sizes where there is sufficient space)

* Consistent with ATAG Report Tool
* Better scannability, because it's the values that matter more than the keys

<h2 id="screenshot">Screenshot</h2>

Two screenshots, the left of current situation, the right of what it looks like with all of the above changed:

<img width="1847" alt="screenshots" src="https://user-images.githubusercontent.com/178782/107500818-ec615400-6b96-11eb-8205-bf1f13d6dc2f.png">

**Comment by nitedog** [comment]
I like this simplification and alignment with ART.

Just making sure that "Report title" and "Record of evaluation specifics (optional)" will be integrated in the updated design too?

**Comment by hidde** [comment]
All of these changes are now in https://github.com/w3c/wcag-em-report-tool/pull/382 ([PR preview](https://deploy-preview-382--wcag-em-report-tool-2021-redesign.netlify.app/evaluation/view-report))

An additional change I made is change what's displayed for fields that were left empty. Was “…”, is now “Not Provided” (as in ATAG Report Tool).

I did not find “Report title” and "Record of evaluation specifics (optional)” in the page, where should they appear?

**Comment by hidde** [comment]
Update: there is a heading for “Record of evaluation specifics (optional)” included in the page. 

The Report Title is not used. I wonder if we need this, or could default to “Report”; people will probably change it anyway when they import the JSON into their own generation tools, or can change it when they download the HTML.

**Comment by nitedog** [comment]
"Report Title" is currently reflected in the preview report (the box below the buttons in the "View Report" step) and in the generated report. We should keep that functionality.

**Comment by hidde** [comment]
Thanks!

In the current prototype there are two `h1`s:

* at the top, I assume this is the page's name, and can be replaced with 'Report', so that it is the same for everyone using this report tool
* a little below the top, I assume this is the report's name / 'report title'; this is different if you changed the report title


In my PR ([preview](https://deploy-preview-382--wcag-em-report-tool-2021-redesign.netlify.app)) I removed this second `h1`, as two `h1`s is hierarchically questionable, and if they are very similar it may be a confusing user experience. 

Now that you explained it I do see the use case for a user generated report title that is different from the preview page's title. On the other hand, ART doesn't have this concept of a second, user generated title, and we did not get comments on that. 

Would it make sense to remove the user generated title from this page, and only display it in the HTML/JSON downloads? 

Or, perhaps better, display it as part of the report's meta data, along with Report Creator and Evaluation Date, so that people can see it as a confirmation?

**Comment by hidde** [comment]
Discussed with Shadi: 

* Record of evaluation specifics can display at the end
* Heading “Report” becomes the report title if it is failed in, otherwise falls back to “Report”

**Comment by hidde** [comment]
Have updated the PR so that it now shows the user generated report title, and falls back to “Report”.
